CHANGELOG.md

```diff
@@ -1,5 +1,7 @@
 # Changelog
 
+* Migrate away from `make` commands in testing
```

Comment:
🎉

Reply:
Joey, could you pls take a look at the (very welcome and excellent!) commits by @paazmaya? The ones that remove make and substitute npm?I think all the tasks from @rowanmanning's [makefile](https://github.com/rowanmanning/make/blob/master/javascript/index.mk) are there but I'm not 100% sure as I'm JUST SO NEW to the codebase.

Reply:
Yup, manually checked and @paazmaya has replicated everything the `make ci` command was doing ⭐ .

Reply:
Wonderful!